CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Minor: Added support for FrankerFaceZ animated emote links. (#455)
 - Fix: 7TV emote images now resolve correctly. (#456)
 - Dev: Only upload Ubuntu binaries to releases. (#454)
+- Dev: Set version in User-Agent. (#459)
```

Comment:
Move this changelog entry to under `## Unreleased` as this change is not part of the v2.0.1 release